easybuild/easyconfigs/h/hwloc/hwloc-1.11.10-GCCcore-7.3.0.eb

```diff
@@ -25,15 +25,20 @@ source_urls = ['https://www.open-mpi.org/software/hwloc/v%(version_major_minor)s
 sources = [SOURCE_TAR_GZ]
 checksums = ['0a2530b739d9ebf60c4c1e86adb5451a20d9e78f7798cf78d0147cc6df328aac']
 
+osdependency = [('libudev-dev', 'libudev-devel')]
+
 builddependencies = [
     ('binutils', '2.30'),
 ]
 
 dependencies = [
     ('numactl', '2.0.11'),
+    ('libxml2', '2.9.8'),
+    ('libpciaccess', '0.14'),
 ]
 
 configopts = "--enable-libnuma=$EBROOTNUMACTL"
```

Comment:
missing space here at the end (same for several other easyconfigs)

Reply:
missing space??

Reply:
Ahh for the configopts... fixing...